### PR TITLE
HPCC-15391 Fix Unexpected error:'<type 'exceptions.UnicodeDecodeError'>' in Regression Test Engine log processor

### PR DESCRIPTION
**Pull request by AttilaVamos**
Fix Unicode character handling.

Add more debug log

Signed-off-by: Attila Vamos attila.vamos@gmail.com

### CODE REVIEW

**Comment by AttilaVamos** [comment]
@richardkchapman please check and merge.

File: testing/regress/hpcc/util/ecl/command.py [hunk 1]
```diff
@@ -157,7 +157,7 @@ def runCmd(self, cmd, cluster, eclfile, report, **kwargs):
                     test = True
                 elif (res['state'] == 'failed') and (('error' in data) or ('exception' in data.lower())):
                     eclfile.diff = ("%3d. Test: %s\n") % (eclfile.taskId, eclfile.getBaseEclRealName())
-                    eclfile.diff += data
+                    eclfile.diff += repr(data).replace("\\xe2\\x80\\x98", "'").replace("\\xe2\\x80\\x99", "'")
```
Comment (richardkchapman) [inline testing/regress/hpcc/util/ecl/command.py]: Would a more general "Convert to ascii ignoring any errors" call here be more appropriate?

Comment (AttilaVamos) [inline testing/regress/hpcc/util/ecl/command.py]: Right

**Comment by AttilaVamos** [comment]
@richardkchapman updated.

**Pull request merged**